### ISSUE TITLE
Redux integration

### ISSUE THREAD

**Issue by JMStudiosJoe** [codeforsanjose/SJPL-MealTally#61]

**Comment by JMStudiosJoe** [comment]
maybe not yet but reviewing this blog https://www.valentinog.com/blog/react-redux-tutorial-beginners/

**Comment by thedocbwarren** [comment]
How about this as an alternative:

https://reactjs.org/docs/context.html

https://medium.com/dailyjs/reacts-%EF%B8%8F-new-context-api-70c9fe01596b

https://medium.freecodecamp.org/replacing-redux-with-the-new-react-context-api-8f5d01a00e8c

**Comment by JMStudiosJoe** [comment]
Ah I heard about this yes will look into it.  But for now need to determine if we need it yet or not and I’m think long we don’t need it yet. What are your thoughts @thedocbwarren ?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by thedocbwarren** [comment]
I don’t think so yetbas there aren’t a lot of components.  Context and and some cool patterns can help keep things neat and don’t require redux or difficult property passing.

**Comment by JMStudiosJoe** [comment]
closing issue since not needed yet

**Comment by JMStudiosJoe** [comment]
ok so think i have hit the mark where I need redux or something. When viewing a report and user wants to edit it from the modal which is quite a few components deep from MealTallyComponent so is https://reactjs.org/docs/context.html what the cool kids on the block are using these days? @thedocbwarren

**Comment by JMStudiosJoe** [comment]
I see their example passing a simple string but what about an object or function like editReport which takes a report object then just goes to the MealTallyComponent?

**Comment by JMStudiosJoe** [comment]
Turns out users will not be editing much and can create a new one to override the one they need to edit but still should think about integrating context or redux.

**Comment by JMStudiosJoe** [comment]
context it is

**Comment by ctram** [comment]
@JMStudiosJoe Hi folks, are you guys accepting contributions for this project?
